shared/Cargo.toml

```diff
@@ -23,8 +26,17 @@ rust_decimal_macros = "1.29.0"
 serde = { version = "1.0.158", features = ["derive"] }
 serde_json = "1.0.93"
 serde_with = "2.3.1"
-sqlx = { version = "0.6", features = [ "offline", "runtime-tokio-rustls", "postgres", "decimal", "uuid", "chrono", "json" ] }
-sqlxmq = { version = "0.4.1", default-features = false, features = [ "runtime-tokio-rustls" ] }
+sqlx = { version = "0.7.1", features = [
+    "runtime-tokio-rustls",
+    "postgres",
+    "rust_decimal",
+    "uuid",
+    "chrono",
+    "json",
+] }
+sqlxmq = { git = "https://github.com/Diggsey/sqlxmq", rev = "658d584", default-features = false, features = [
```

Comment:
we forked this for bria. I think we should use it here. 
`sqlxmq = { git = "https://github.com/GaloyMoney/sqlxmq", rev = "f03f73c", default-features = false, features = ["runtime-tokio-rustls"] }`